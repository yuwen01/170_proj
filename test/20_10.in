20 
99
0 1 75.127 8.619
0 2 82.175 7.483
0 3 99.141 7.025
0 4 54.328 4.425
0 5 84.724 9.417
0 6 36.313 2.012
0 7 67.869 5.051
0 8 6.327 1.747
0 9 31.791 0.005
0 10 0.998 7.382
0 11 12.427 2.853
0 12 32.168 5.420
0 13 16.682 7.423
0 14 50.169 0.547
0 15 4.744 5.070
0 16 70.742 4.361
0 17 95.066 1.540
0 18 41.285 1.101
0 19 5.137 3.761
1 2 80.930 0.702
1 3 93.739 3.533
1 4 38.769 0.720
1 5 12.023 0.550
1 6 13.941 8.762
1 7 41.372 9.613
1 8 41.319 4.625
1 9 99.438 2.655
1 10 73.223 7.250
1 11 49.804 0.990
1 12 45.763 7.607
1 13 59.365 8.478
1 14 50.107 6.307
1 15 2.350 4.488
1 16 90.265 5.060
1 17 2.200 9.317
1 18 98.710 1.344
1 19 58.615 3.721
2 3 68.150 4.797
2 4 73.584 2.650
2 5 70.660 3.566
2 6 67.530 8.712
2 7 6.499 2.214
2 8 81.520 6.983
2 9 90.399 4.098
2 10 98.190 3.730
2 11 26.908 5.700
2 12 38.572 3.279
2 13 33.154 8.417
2 14 34.255 4.087
2 15 10.005 3.201
2 16 34.451 6.989
2 17 47.243 5.583
2 18 56.966 5.027
2 19 74.177 8.090
3 4 2.217 1.933
3 5 36.238 2.967
3 6 0.526 2.817
3 7 70.133 6.465
3 8 50.523 8.435
3 9 84.244 0.468
3 10 81.783 1.358
3 11 55.569 0.135
3 12 87.569 4.638
3 13 83.557 3.906
3 14 44.082 2.580
3 15 62.640 9.541
3 16 10.711 4.269
3 17 0.232 5.134
3 18 55.959 1.567
3 19 71.545 5.627
4 5 48.556 0.650
4 6 33.656 4.928
4 7 34.057 3.605
4 8 78.427 6.335
4 9 9.767 8.282
4 10 48.167 0.179
4 11 17.187 1.390
4 12 61.150 1.452
4 13 88.922 9.935
4 14 6.681 6.745
4 15 25.920 0.796
4 16 21.918 8.533
4 17 1.226 6.600
4 18 28.365 3.367
4 19 60.070 7.028
5 6 12.558 4.119
5 7 91.444 8.928
5 8 88.519 6.359
5 9 20.606 9.080
5 10 23.439 7.669
5 11 86.383 6.480
5 12 30.693 0.132
5 13 60.031 2.767
5 14 97.736 0.043
5 15 80.744 7.477
5 16 47.193 6.332
5 17 57.320 0.245
5 18 50.395 8.492
5 19 76.553 9.540
6 7 97.277 9.049
6 8 75.982 0.751
6 9 52.496 7.674
6 10 56.836 8.858
6 11 79.015 0.981
6 12 14.994 6.706
6 13 50.007 9.144
6 14 30.822 1.819
6 15 32.747 4.766
6 16 48.164 7.363
6 17 83.218 4.099
6 18 41.027 3.472
6 19 42.623 0.711
7 8 63.879 8.308
7 9 41.123 4.251
7 10 6.653 3.124
7 11 94.294 6.729
7 12 27.286 5.872
7 13 24.884 0.994
7 14 43.012 8.410
7 15 13.431 3.100
7 16 49.942 5.694
7 17 86.761 2.415
7 18 62.748 9.689
7 19 39.589 7.724
8 9 78.148 2.042
8 10 70.696 1.284
8 11 47.360 4.164
8 12 12.820 1.852
8 13 32.600 1.013
8 14 6.052 2.736
8 15 3.335 1.244
8 16 30.363 9.239
8 17 52.918 4.531
8 18 45.810 5.391
8 19 31.332 2.729
9 10 41.512 0.709
9 11 55.556 0.853
9 12 21.814 1.967
9 13 70.468 4.719
9 14 25.836 8.393
9 15 69.308 2.951
9 16 27.244 3.858
9 17 69.787 5.912
9 18 44.933 7.531
9 19 45.923 7.321
10 11 38.121 5.271
10 12 69.573 3.678
10 13 16.729 8.170
10 14 89.666 6.266
10 15 88.338 0.233
10 16 73.017 2.308
10 17 27.863 3.087
10 18 65.664 5.079
10 19 1.434 0.538
11 12 67.425 6.638
11 13 32.622 6.076
11 14 85.437 5.696
11 15 86.067 9.027
11 16 29.723 6.486
11 17 33.763 2.813
11 18 52.363 5.021
11 19 93.139 0.166
12 13 83.605 7.888
12 14 38.349 3.876
12 15 70.205 5.270
12 16 49.942 8.340
12 17 2.611 0.126
12 18 73.898 9.297
12 19 19.440 7.784
13 14 76.423 5.771
13 15 85.855 2.701
13 16 75.330 7.473
13 17 40.911 7.351
13 18 58.198 1.865
13 19 52.200 8.328
14 15 54.798 6.622
14 16 34.899 8.112
14 17 27.153 3.360
14 18 0.453 0.259
14 19 46.348 2.787
15 16 51.847 1.154
15 17 79.922 4.946
15 18 98.582 0.667
15 19 70.310 9.588
16 17 49.487 2.999
16 18 2.036 0.048
16 19 42.195 2.756
17 18 29.314 9.970
17 19 5.306 3.555
18 19 54.200 2.710
