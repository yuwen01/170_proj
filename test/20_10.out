0 17
1 9
2 14
3 9
4 8
5 14
6 8
7 17
8 8
9 9
10 18
11 9
12 13
13 13
14 14
15 18
16 18
17 17
18 18
19 19
